### DISCUSSION TITLE
Using PFC 2022 n PB2022 Build1900 Menu disappear

### DISCUSSION THREAD

**Discussion by jojo5791** [OpenSourcePFCLibraries/2022#3] [Q&A]
Hi Master,
   I urgently needed assistance on the configuration of pfc security. The menu went haywired (see picture). "Before" picture is the original. "After" picture is the after configuration. The other of icon/menu also went missing after i invisible m_operations only.  "Settings" picture is to show you what i had set. 
 "Weird" picture is the second issue, this error pop when i click save. And you noticed there is a blank row with 'Enabled' setting. 
Please help. Thank you very much
![before](https://github.com/OpenSourcePFCLibraries/2022/assets/70992666/26ade697-b550-4e5a-a24d-5bf437e435c9)
![after](https://github.com/OpenSourcePFCLibraries/2022/assets/70992666/f27a5a82-8ea6-48fa-8fe6-e4d8f4fca996)
![setting](https://github.com/OpenSourcePFCLibraries/2022/assets/70992666/438f7a39-3423-4611-8461-91a3023ab348)
![weird](https://github.com/OpenSourcePFCLibraries/2022/assets/70992666/91080f30-e617-4d82-86dd-5a8d359e7bfd)

**Comment by bruce-armstrong** [comment]
Sorry, but with the preparation for Elevate and dealing with items coming out of it I'm just getting around to this.  Apologize for that.

It there any way you could provide a sample of what you're running into that I can run locally?  Perhaps the code for your app if you can share that somehow.  Perhaps I could disable login and database access enough to see what appears when you first run app without needed database.
